src/js/view/reducers/snackBarReducer.js

```diff
@@ -1,22 +1,29 @@
-const initialState = { show: false, message: '', level: 'info', fadeAfter: 0 };
+const initialState = {
+  index: 0,
+  show: false,
+  message: '',
+  level: 'info'
+};
+
+const nextIndex = index => {
+  return ~~index + 1;
```

Comment:
Varför behövs dubbel NOT här?

Reply:
Det är inte nödvändigt. Effekten av det är att det går att hantera icke-int-värden som undefined eller NaN som exempelvis tolkas som 0. Men det kanske är ett beteende som leder till mer buggar än vad det fixar?

Reply:
Ok då förstår jag. Jag tror inte att index kan bli undefined eller NaN eftersom initialState sätter den till 0 och den ökar bara med heltal efter det men jag tror inte att den kommer att leda till buggar heller eftersom den ökar med heltal bara och inte decimaler.

Reply:
Bör jag ta bort det? Det kanske bara är förvirrande att se det i koden, om resten av systemet är uppsatt för att aldrig få den situationen att uppstå-

Reply:
Jag tog bort det i senaste committen. Eftersom vi har den här diskussionen till att börja med visar bara att det gav förvirring och att det inte fixar något nu existerande problem. :smile:

Reply:
👍 ok